src/Phinx/Db/Adapter/SQLiteAdapter.php

```diff
@@ -73,7 +73,7 @@ public function connect()
             $options = $this->getOptions();
 
             // if port is specified use it, otherwise use the MySQL default
-            if (isset($options['memory'])) {
+            if (isset($options['memory']) && $options['memory']) {
```

Comment:
Why that? If so, you could use !empty() as one call.

Reply:
Sure. `empty()` isn't in my mental toolset, so I didn't think of it. I'll make the change.

Reply:
I think I just had a similar issue with getenv() being false instead of null ( https://github.com/cakephp/cakephp/pull/13242 ), running into similar issues if you use too strict check on the null type.